src/Stratis.Bitcoin.Features.MemoryPool.Tests/TestChainFactory.cs

```diff
@@ -77,16 +78,20 @@ public static async Task<ITestChainContext> CreateAsync(Network network, Script
             network.Consensus.Options = new ConsensusOptions();
             new FullNodeBuilderConsensusExtension.PowConsensusRulesRegistration().RegisterRules(network.Consensus);
 
+            // Dont check PoW of a header in this test.
+            IHeaderValidationConsensusRule checkDiffRule = network.Consensus.HeaderValidationRules.SingleOrDefault(x => x.GetType() == typeof(CheckDifficultyPowRule));
+            network.Consensus.HeaderValidationRules.Remove(checkDiffRule);
```

Comment:
Could do network.Consensus.HeaderValidationRules.RemoveAll(x => x.GetType() == typeof(CheckDifficultyPowRule)); here.

Reply:
fixed

Reply:
yes. but that will also require 2 lines. one liner is shorter.